### PR TITLE
Fixed footer copyright / fixed rawhide link

### PR DESCRIPTION
**Pull request by dyoungwd**
- updated footer copyright to 2019 
-fixed rawhide download link ( was 404 ) and updated to correct checksum

### CODE REVIEW

**Comment by pnemade** [comment]
Same here, commits can be squashed into a single commit.

File: pages/download/index.html [hunk 1]
```diff
@@ -37,12 +37,12 @@ <h1>Try the latest</h1>
 <p>Help us make Silverblue better by testing the <b>Rawhide</b> builds of Silverblue that
 will eventually become Fedora 31.</p>
 
-<p><a class="button" href='https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-Rawhide-20190428.n.0.iso'><svg class="icon"><use href="/public/svg-sprites.svg#download" /></svg>64-bit 2.1GB Silverblue image (Fedora Rawhide)</a></p>
+<p><a class="button" href='https://www.mirrorservice.org/sites/dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-Rawhide-20190504.n.0.iso'><svg class="icon"><use href="/public/svg-sprites.svg#download" /></svg>64-bit 2.1GB Silverblue image (Fedora Rawhide)</a></p>
```
Comment (miabbott) [inline pages/download/index.html]: I'd rather not see us rely on an external service for pointing at the ISO; let's just keep `download.fedoraproject.org` and update the ISO used:

https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-Rawhide-20190501.n.0.iso

File: pages/download/index.html [hunk 2]
```diff
@@ -37,12 +37,12 @@ <h1>Try the latest</h1>
...
 
-<p> You can verify your download with 
-<a href='https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-Rawhide-x86_64-20190428.n.0-CHECKSUM'>
+<p> You can verify your download with
+<a href='https://www.mirrorservice.org/sites/dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-Rawhide-x86_64-20190504.n.0-CHECKSUM'>
```
Comment (miabbott) [inline pages/download/index.html]: See above comment about ISO

File: pages/download/index.html (hunk 1 above)
Comment (dyoungwd) [reply pages/download/index.html]: That will be https://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/ and same for iso

File: pages/download/index.html (hunk 2 above)
Comment (dyoungwd) [reply pages/download/index.html]: See above

**Comment by miabbott** [comment]
Looks like there are some conflicts.  Could you rebase your commits onto `master`?

This worked for me `git pull -r upstream master`

**Comment by dyoungwd** [comment]
Will do in the next hour

**Comment by dyoungwd** [comment]
That command not working but showing no conflicts at the moment with base branch

**Comment by miabbott** [comment]
GitHub is still complaining on this end; won't let me merge and also won't show me the conflicts...

```
This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch. 
```

**Comment by dyoungwd** [comment]
Should be it rebased now I think 

`
`

**Pull request closed without merging**